apps/documentation/src/routes/documentation/contributing/local-development.mdx

```diff
@@ -0,0 +1,122 @@
+import MetaTags from '../../../components/meta-tags'
+
+<MetaTags
+  title="Tuono - Contributing - Local development"
+  canonical="https://tuono.dev/documentation/contributing/local-development"
+  description="The project is massive - if you like it, do consider contributing!"
+/>
+
+import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
+
+<Breadcrumbs
+  breadcrumbs={[
+    { label: '✨ Contributing', href: '/documentation/contributing' },
+    { label: 'Local development' },
+  ]}
+/>
+
+# Local development
+
+Thanks for your interest! In this page, you find the instructions to set up the `tuono`!
+
+## Quick setup
```

Comment:
I don't get the benefit of having both "Setup" and "Quick Setup". 
Aren't we giving the same value twice?

Reply:
I was thinking about avoiding to have an "advanced user" scroll all the short explanation of each installation. 
But re-reading made me realise it is not worth it.